packages/yew/Cargo.toml

```diff
@@ -25,16 +25,23 @@ slab = "0.4"
 wasm-bindgen = "0.2"
 yew-macro = { version = "^0.19.0", path = "../yew-macro" }
 thiserror = "1.0"
-futures = { version = "0.3", default-features = false, features = ["std", "async-await"] }
+futures = { version = "0.3", default-features = false, features = ["std"] }
 html-escape = { version = "0.2.9", optional = true }
 implicit-clone = { version = "0.3", features = ["map"] }
 base64ct = { version = "1.5.0", features = ["std"], optional = true }
 bincode = { version = "1.3.3", optional = true }
 serde = { version = "1", features = ["derive"] }
 tracing = "0.1.36"
-pin-project = "1.0.11"
+prokio = "0.1.0"
 rustversion = "1"
 
+[target.'cfg(target_arch = "wasm32")'.dependencies]
+wasm-bindgen-futures = "0.4"
```

Comment:
Why do we still need a direct dependency on this?

Reply:
This is due to yew is using wasm_bindgen_futures::JsFuture in a couple places.